examples/http-header-filter/README.md

```diff
@@ -1,13 +1,13 @@
 # Example
 
 In this example we will deploy NGINX Kubernetes Gateway and configure traffic routing for a simple echo server.
-We will use `HTTPRoute` resources to route traffic to the echo server, using the RequestHeaderModifier filter to modify 
+We will use `HTTPRoute` resources to route traffic to the echo server, using the `RequestHeaderModifier` filter to modify 
```

Comment:
I'd remove backtick for HTTPRoute (it's a resource), while retaining the one RequestHeaderModifier (it is a field value).